### ISSUE TITLE
Talks system: multi-column layout a bit wonky

### ISSUE THREAD

**Issue by koalie** [w3c/design#3]
The multi-column layout doesn’t work well in case the event has a long name because of the long URL, e.g. the entry for [global accessibility awareness day gaad](https://www.w3.org/blog/talks/event/global-accessibility-awareness-day-gaad/) (see screenshot at bottom).

In comparison, the entry for [webvr panel](https://www.w3.org/blog/talks/event/webvr-panel-at-vrx-europe-conference/) works slightly better.

The friend of mine who reported this suggested that instead of 

```html
<a href="https://www.eventbrite.com/e/siteimprove-invites-you-global-accessibility-awareness-day-guest-speaker-sir-tim-berners-lee-tickets-33304300058"   
target="_self">https://www.eventbrite.com/e/siteimprove-invites-you-global-accessibility-awareness-day-guest-speaker-sir-tim-berners-lee-tickets-33304300058</a>
```

A possible fix could be:

```html
<a href="https://www.eventbrite.com/e/siteimprove-invites-you-global-accessibility-awareness-day-guest-speaker-sir-tim-berners-lee-tickets-33304300058" target="_self">EventBrite</a>
```

But I don’t know how wordpress generates that bit and whether the in-your-face URLs can be avoided at all.

 ![w3c-gaad-fullpage](https://user-images.githubusercontent.com/3716853/27385400-d9acbdf8-5692-11e7-97b6-be17ab54d32e.png)

**Comment by tripu** [comment]
[Edited to improve formatting of HTML chunks.]

**Comment by tripu** [comment]
I don't quite like the idea of not showing the URL, but there are times when long URLs are unmanageable.

I'll see if we can trim long URLs down to the domain name, and (perhaps) show the complete URL as a tooltip.

**Comment by tripu** [comment]
(Fixed by adding

```css
.tribe-events-event-url {
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
}
```

to [`/wp-content/themes/w3cblog-child/style.css`](https://www.w3.org/blog/talks/wp-content/themes/w3cblog-child/style.css))
